src/clients/localnet.rs

```diff
@@ -0,0 +1,196 @@
+// Solana localnet client for testing against local Solana test validator
+use crate::solana::{
+    base_client::SolanaBaseClient,
+    rpc_client::SolanaRpcClient,
+    signing_client::{SolanaSigningClient, SolanaSigningClientImpl},
+};
+use async_trait::async_trait;
+use solana_sdk::{
+    commitment_config::CommitmentConfig,
+    signature::Keypair,
+};
+
+/// Solana localnet client for development and testing
+pub struct SolanaLocalnetClient {
```

Comment:
is there anything different about solana localnet from mainnet? ideally we wouldn't distinguish the two in code, but rather just configure them with differently

Reply:
fixed in latest commit